### PR TITLE
New tiers post

### PR DESCRIPTION
**Pull request by beta-ziliani**

### CODE REVIEW

File: _posts/2024-03-25-new-sponsoring-tiers.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The sponsoring tiers and their perks hadn't been updated in a while and we felt it was time to freshen up.
+Particularly, we want to offer more differentiated opportunities for expressing thanks to our sponsors.
+
+The general structure of tiers hasn't changed. It's a bit more refined now with clearer naming.
+We consider three groups of sponsorship levels with different target audiences:
+
+1. **Backer:** Individuals and small organizations who love Crystal and want to support it.
+2. **Sponsor:** Individuals and organizations who want to support Crystal and maybe gain some visibility.
+3. **Corporate:** Organizations who want to support Crystal and gain support to move their business forward.
```
Comment (beta-ziliani) [inline _posts/2024-03-25-new-sponsoring-tiers.md:17]: If the label is singular, keep it singular.

```suggestion
1. **Backer:** An individual or small organization who loves Crystal and wants to support it.
2. **Sponsor:** An individual or organization who wants to support Crystal and maybe gain some visibility.
3. **Corporate:** An organization who wants to support Crystal and obtain support to move their business forward.
```

Comment (beta-ziliani) [reply _posts/2024-03-25-new-sponsoring-tiers.md:17]: Alternatively:

```suggestion
1. **Backers:** Individuals and small organizations who love Crystal and want to support it.
2. **Sponsors:** Individuals and organizations who want to support Crystal and maybe gain some visibility.
3. **Corporates:** Organizations who want to support Crystal and obtain support to move their business forward.
```

File: _posts/2024-03-25-new-sponsoring-tiers.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  * Small logo at *Silver Sponsor* ($350)
+  * Big logo at *Gold Sponsor* ($750)
+4. Directed Corporate Sponsors to contact us to better tailor a package that suits their needs.
+
+Everyone who is already a sponsor stays in the tier that corresponds to their current monthly contribution.
+Some of the perks in that tier may have changed. We understand this is a breaking change.
+If this affects you, please let us know at [crystal@manas.tech](mailto:crystal@manas.tech).
+
+The new perks will take a couple of days to take effect.
+We need some time to properly update everything.
```
Comment (beta-ziliani) [inline _posts/2024-03-25-new-sponsoring-tiers.md:37]: I don't think this adds much.

```suggestion
```

File: _posts/2024-03-25-new-sponsoring-tiers.md (hunk 1 above)
Comment (beta-ziliani) [reply _posts/2024-03-25-new-sponsoring-tiers.md:17]: (I also changed `gain` for `obtain`. It can also work with `get`.)

**Review by beta-ziliani — COMMENTED** [review]
I love the changes, thanks @straight-shoota ! Just a few remarks.

File: _posts/2024-03-25-new-sponsoring-tiers.md (hunk 1 above)
Comment (straight-shoota) [reply _posts/2024-03-25-new-sponsoring-tiers.md:17]: I actually don't see any dissonance here: "The Backer tier consists of individuals and organizations." That's totally legit grammar and IMO makes the most sense.

The singular description feels awkward. The singular labels are derived from the tier names, which fits very well.

File: assets/blog/2024/2024-03-26-open-collective-corporate.webp
Comment (beta-ziliani) [inline assets/blog/2024/2024-03-26-open-collective-corporate.webp:1]: The Donation is not part of the Corporate. We can add one more level "One-timers", and advertise it as for either people who want to do a one-time contribution, or for those needing one-time support.

File: _posts/2024-03-25-new-sponsoring-tiers.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+We made several changes to the sponsor packages offered on the [OpenCollective sponsoring page](https://www.opencollective.com/crystal-lang).
+
+The sponsoring tiers and their perks hadn't been updated in a while and we felt it was time to freshen up.
+Particularly, we want to offer more differentiated opportunities for expressing thanks to our sponsors.
+
+The general structure of tiers hasn't changed. It's a bit more refined now with clearer naming.
+We consider three groups of sponsorship levels with different target audiences:
+
+1. **Backer:** Individuals and small organizations who love Crystal and want to support it.
```
Comment (beta-ziliani) [inline _posts/2024-03-25-new-sponsoring-tiers.md:15]: The images are a bit bulky and disrupt the text. Maybe trim the cards?
image

Or even more to stick to the essentials?
image

**Comment by beta-ziliani** [comment]
Ouch
image

**Comment by straight-shoota** [comment]
Yeah the fun typography of the old website ... 🙈

**Review by beta-ziliani — COMMENTED** [review]
Glad you find it! OK, this is ready to 🚀

**Pull request merged**